experimenter/experimenter/settings.py

```diff
@@ -487,15 +487,31 @@
 KINTO_REVIEW_TIMEOUT = config("KINTO_REVIEW_TIMEOUT", cast=int)
 
 # Jetstream GCS Bucket data
-DEFAULT_FILE_STORAGE = "storages.backends.gcloud.GoogleCloudStorage"
+# DEFAULT_FILE_STORAGE = "storages.backends.gcloud.GoogleCloudStorage"
```

Comment:
Should we remove this line or leave it commented?

Reply:
Oops!